flashinfer_bench/data/validate.py

```
@@ -0,0 +1,1245 @@
+        traces={},
+    )
+
+    config = BenchmarkConfig(
+        warmup_runs=2, iterations=5, num_trials=1, definitions=[definition_name]
+    )
+
+    try:
+        benchmark = Benchmark(trace_set, config)
+        result_trace_set = benchmark.run_all(dump_traces=False)
+        benchmark.close()
+    except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)